### DISCUSSION TITLE
Too much whitespace on the left

### DISCUSSION THREAD

**Discussion by jan-vandenberg** [tooot-app/app#555] [Ideas]
I feel there is too much whitespace on the left: that costs screen realestate.

Personally I'd like to this space used (just like the official Mastodon app) or maybe have an UI option for people that want to use this:
- e.g. make toots tighter.

![image](https://user-images.githubusercontent.com/13764557/206237621-4293f99a-0d8e-409b-af0d-94b1234f7850.png)

**Comment by xmflsct** [comment]
ehh, yeah..

**Comment by jan-vandenberg** [comment]
Someone on Mastodon pointed this out to me, and now I can't unsee it 🙈 

https://mastodon.social/@raaphorst/109431543474120852

**Comment by xmflsct** [comment]
haha, just need to convince your mind again that, Twitter also does this! 😉
